src/components/Tooltip/Tooltip.vue

```diff
@@ -0,0 +1,105 @@
+<template>
+    <div
+        ref="tooltip"
+        class="d-inline-block"
+    >
+        <slot v-if="!!$slots.default" />
```

Comment:
Unsure but I think there is no need for this.
```suggestion
        <slot />
```